backend/src/v5/processors/teamspaces/teamspaces.js

```diff
@@ -67,4 +70,28 @@ Teamspaces.getTeamspaceMembersInfo = async (teamspace) => {
 	);
 };
 
+Teamspaces.getQuotaInfo = async (teamspace) => {
+	try {
+		const quotaInfo = await getQuotaInfo(teamspace, true);
+		const spaceUsed = await getSpacedUsed(teamspace, true);
+		const collaboratorsUsed = await getCollaboratorsUsed(teamspace);
+
+		return {
+			data: { available: quotaInfo.quota, used: spaceUsed },
+			seats: { available: quotaInfo.collaboratorLimit, used: collaboratorsUsed },
+		};
+	} catch {
+		// exception is thrown if there is no valid subscription
```

Comment:
its because this method can throw an exception at line 53 :/

![image](https://user-images.githubusercontent.com/71635617/173823954-655aee8e-5767-4eab-8feb-110acc448c1a.png)

Reply:
let it throw, it's suppose to throw

Reply:
thats different than v4 though right ? cause in v4 if you dont have a valid subscription it just returns 0s